lib/suse_container_urls.pm

```diff
@@ -75,20 +75,16 @@ sub get_suse_container_urls {
     }
     elsif (is_tumbleweed || is_microos("Tumbleweed")) {
         push @image_names,  "registry.opensuse.org/" . get_opensuse_registry_prefix . "opensuse/tumbleweed";
-        push @stable_names, "docker.io/opensuse/tumbleweed";
```

Comment:
Deleted without replacement?

Reply:
What should be the replacement? I thought we rely on registry.opensuse.org only for opensuse images

Reply:
The replacement would be to use `registry.opensuse.org` instead of `docker.io` here, instead of removing it.

Reply:
Note that `image_names` is "before testing" and `stable_names` is "after testing".